internal/store/pod_test.go

```diff
@@ -1072,6 +1074,28 @@ kube_pod_container_status_last_terminated_reason{container="container7",namespac
 			`,
 			MetricNames: []string{"kube_pod_status_scheduled", "kube_pod_status_scheduled_time"},
 		},
+		{
+			Obj: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "pod2",
+					Namespace: "ns2",
+				},
+				Status: v1.PodStatus{
+					Conditions: []v1.PodCondition{
+						{
+							Type:    v1.PodScheduled,
+							Status:  v1.ConditionFalse,
+							Reason:  "Unschedulable",
```

Comment:
Is this the only known value for `reason`?

Reply:
yes, according to https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L2213, `Unschedulable`  is the only known reason for this situation.